README.md

```diff
@@ -18,7 +18,7 @@ Browsing the [sample app](./XamarinCommunityToolkitSample) is the best place to
 
 The toolkit is available via NuGet, and should be installed into all your projects:
 
-* NuGet Official Releases: [![NuGet](https://img.shields.io/nuget/vpre/Xamarin.CommunityToolkit.svg?label=NuGet)](https://www.nuget.org/packages/Microsoft.Toolkit.Xamarin.Forms)
+* NuGet Official Releases: [![NuGet](https://img.shields.io/nuget/vpre/Xamarin.CommunityToolkit.svg?label=NuGet)](https://www.nuget.org/packages/Xamarin.CommunityToolkit)
```

Comment:
Should these lines be discarded now?

Reply:
Yeah. I closed this PR as another was created fixing this. The other one doesn’t have this line anymore.

Reply:
Yeah. Sorry about that, another pull request fixed this so I closed the pull request with this commit but my branch still had this apparently.

Reply:
Can you please fix the merge conflict? (You can remove this commit or just merge the latest "main" code)